iceddev/git-splainin
# add a release process

#### What's this PR do?

Adds a release process to travis.
#### Where should the reviewer start?

The zipping is done in the gulpfile and the release process is configured in the travis.yml
#### How should this be manually tested?

Run `npm run release` and unzip it.  Then load it in the browser and make sure the application still works
#### Any background context you want to provide?

Need an easy way to release this.
